docs/package.json

```diff
@@ -8,10 +8,12 @@
     "build": "cross-env NODE_ENV=production next build --profile",
     "build:clean": "rimraf .next && yarn build",
     "build-sw": "node ./scripts/buildServiceWorker.js",
-    "dev": "rimraf ./node_modules/.cache/babel-loader && next dev --port 3001",
```

Comment:
No longer needed since we are no longer using https://github.com/kentcdodds/babel-plugin-macros.